### PR TITLE
add method to unsubscribe by filter and callback

### PR DESCRIPTION
**Pull request by sashless**

### CODE REVIEW

File: chromeps.js [hunk 1]
```diff
@@ -123,12 +123,25 @@ module.exports = (function() {
...
+    var index = callbacks[filter].indexOf(callback);
+    if(index === -1){
+      throw new Error('Callback not registered for filter ' + filter);
+    }
+
+    // callback processing is async and splicing the callback out of the array
+    // while its processed would result in errors
+    // We could also set it to false and then check for a valid function
+    // but i think thats the best way
+    callbacks[filter][index] = function() {};
```
Comment (rwos) [inline chromeps.js]: But if you subscribe and then unsubscribe and subscribe the next thing, you're always increasing the size of this array? That doesn't seem too wise.

Comment (sashless) [inline chromeps.js]: Suggestions?

Richard Wossal <notifications@github.com> schrieb am Mo., 18. Apr. 2016
16:16:

Comment (rwos) [inline chromeps.js]: Maybe [].forEach() works instead of the for loop above?

Comment (sashless) [inline chromeps.js]: forEach seems to work

**Pull request merged**